### PR TITLE
RavenDB-13061: ConnectionStringOperation should expect for RaftComman…

### PR DESCRIPTION
**Pull request by garayx**
…dIndex

### CODE REVIEW

File: src/Raven.Client/Documents/Operations/ConnectionStrings/PutConnectionStringOperation.cs [hunk 1]
```diff
@@ -68,6 +68,9 @@ public override void SetResponse(JsonOperationContext context, BlittableJsonRead
 
     public class PutConnectionStringResult
     {
+        [Obsolete("Not used to get operation result. Use RaftCommandIndex instead")]
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Operations/ConnectionStrings/PutConnectionStringOperation.cs]: `[Obsolete("PutConnectionStringResult.ETag is not supported anymore. Will be removed in next major version of the product. Please use PutConnectionStringResult.RaftCommandIndex instead.")]`

File: src/Raven.Client/Documents/Operations/ConnectionStrings/RemoveConnectionStringOperation.cs [hunk 2]
```diff
@@ -59,6 +59,9 @@ public override void SetResponse(JsonOperationContext context, BlittableJsonRead
 
     public class RemoveConnectionStringResult
     {
+        [Obsolete("Not used to get operation result. Use RaftCommandIndex instead")]
```
Comment (ppekrol) [inline src/Raven.Client/Documents/Operations/ConnectionStrings/RemoveConnectionStringOperation.cs]: `[Obsolete("RemoveConnectionStringResult.ETag is not supported anymore. Will be removed in next major version of the product. Please use RemoveConnectionStringResult.RaftCommandIndex instead.")]`

**Review by ppekrol — COMMENTED** [review]
Can you add some tests for this or add assertion to existing test at least?

File: src/Raven.Client/Documents/Operations/ConnectionStrings/PutConnectionStringOperation.cs (hunk 1 above)
Comment (garayx) [reply src/Raven.Client/Documents/Operations/ConnectionStrings/PutConnectionStringOperation.cs]: done

File: src/Raven.Client/Documents/Operations/ConnectionStrings/RemoveConnectionStringOperation.cs (hunk 2 above)
Comment (garayx) [reply src/Raven.Client/Documents/Operations/ConnectionStrings/RemoveConnectionStringOperation.cs]: done

**Review by ppekrol — APPROVED** [review]

**Pull request merged**